brd-ios/breadwallet/src/ViewControllers/HomeScreenViewController.swift

```diff
@@ -97,12 +97,12 @@ class HomeScreenViewController: UIViewController, Subscriber {
     }
     
     @objc func reload() {
-        setupSubscriptions()
```

Comment:
we sure?

Reply:
does it do anything? 🤔 
subscribers are never removed afaik... so there shouldn't be a need to resubscribe 🤔

Reply:
will re-add in a future PR (just 2 be safe)